core/client/src/main/java/alluxio/client/block/BlockWorkerClient.java

```diff
@@ -219,6 +219,16 @@ private synchronized void connectOperation() throws IOException {
         LOG.error(e.getMessage(), e);
         return;
       }
+
+      // Send a heartbeat to the worker to register the new session id
+      try {
+        mClient.sessionHeartbeat(mSessionId, mClientMetrics.getHeartbeatData());
+      } catch (Exception e) {
+        LOG.error(e.getMessage(), e);
```

Comment:
the first argument is redundant since the stack trace will already contain `e.getMessage()`. maybe change to something like "failed to send worker session heartbeat"